internal/cluster/transition.go

```diff
@@ -630,6 +642,66 @@ func (th *transitionHandler) PostRefreshCluster(reason string) stateswitch.PostT
 	return ret
 }
 
+func FinalizingStage(sCluster *stateCluster) interface{} {
+	return sCluster.cluster.Progress.FinalizingStage
+}
+
+func (th *transitionHandler) finalizingStageTimeoutMinutes(sCluster *stateCluster) int64 {
+	return int64(finalizingStageTimeout(sCluster.cluster.Progress.FinalizingStage, sCluster.cluster.MonitoredOperators, th.log).Minutes())
+}
+
+func (th *transitionHandler) FinalizingStageTimeoutMinutes(sCluster *stateCluster) interface{} {
+	return th.finalizingStageTimeoutMinutes(sCluster)
+}
+
+func (th *transitionHandler) PostRefreshFinalizingStageTimedOut(sw stateswitch.StateSwitch, args stateswitch.TransitionArgs) error {
```

Comment:
```suggestion
func (th *transitionHandler) PostRefreshFinalizingStageSoftTimedOut(sw stateswitch.StateSwitch, args stateswitch.TransitionArgs) error {
```

Just to be clear that this isn't the real timeout.

Reply:
OK